pkg/server/smb/server.go

```diff
@@ -87,6 +87,7 @@ func (s *Server) NewSmbGlobalMapping(context context.Context, request *internal.
 		}
 	}
 
+	klog.V(2).Infof("done calling NewSmbGlobalMapping")
```

Comment:
it's better set like this:
```
done calling NewSmbGlobalMapping with remote path %q, ...
```

Reply:
@andyzhangx Thanks for the suggestion i will modify this as suggested.

Reply:
@andyzhangx Done. PTAL!